modin/pandas/test/test_io.py

```diff
@@ -1281,6 +1281,25 @@ def test_read_parquet(self, make_parquet_file, columns):
             columns=columns,
         )
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
+        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
+    )
+    def test_read_parquet_indexing_by_column(self, make_parquet_file):
+        # Test indexing into a column of modin with various parquet file row lengths.
+        # Specifically, tests for https://github.com/modin-project/modin/issues/3527
+        # which fails when min_chunk_size (32) < nrows <= min_chunk_size * (num_partitions - 1)
+
+        unique_filename = get_unique_filename(extension="parquet")
+        from modin.config import NPartitions
+
+        nrows = (NPartitions.get() - 1) * 32 - 1
```

Comment:
Could you please:

- Use `MinPartitionSize` instead of 32
- add an assertion that `nrows > MinPartitionSize.get()`? If at some point in the future, `NPartitions` becomes 2,  this will test the case where `nrows = 31 < min_chunk_size = 32` instead of the one you're trying to test.

Reply:
I changed it to use `MinPartitionSize`. I also removed the dependency on `NPartitions`, since `MinPartitionSize + 1` is a more robust failure case i.e. is expected to fail (without these changes) regardless of the value of `MinPartitionSize` or `NPartitions`.

Reply:
Alright!